CMake/External_OpenCV.cmake

```diff
@@ -53,6 +87,7 @@ else()
   list(APPEND OpenCV_EXTRA_BUILD_FLAGS -DWITH_EIGEN:BOOL=FALSE)
```

Comment:
Perhaps I don't understand the OpenCV configuration, but shouldn't WITH_EIGEN=TRUE if we've enabled Eigen? I realize this code is existing, but would be good to make sure it's correct.